app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -319,10 +342,12 @@ def add_proofing_costs(results)
           add_cost(:aamva, transaction_id: hash[:transaction_id])
           track_aamva
         elsif stage == :threatmetrix
-          # transaction_id comes from request_id
```

Comment:
can we keep this comment? it helps when tracing/translating values from partner APIs

Reply:
Ah, I missed that a similar comment is present in the other branches of this if